### ISSUE TITLE
License Consideration 

### ISSUE THREAD

**Issue by aromatic-toast** [UBC-MDS/ggexpress#88]
It should be decided whether we will keep our current license (MIT License) or does our project need some other license? 

## Permissions 
The current license allows for the following: 
- Anyone can use our software for any purpose 
- Anyone could modify , redistribute or sell the work based on our software

## Conditions
The only conditions are that the MIT license and our copyright has to be carried forward on any derivatives of our software. 

## Limitations
We do not have any liability from the software nor do we provide any warranty on the software. 

Is this still an appropriate license for us considering the project? Please provide your thoughts below.

**Comment by zouwenjiao** [comment]
I agree with this license since the purpose of our package is to provide new functions for user's convenience and we also encourage contributions from others. The MIT license coordinates with our package.

**Comment by jacktan1** [comment]
I think the current MIT license is fine since our work is not proprietary and we want everyone to be able to use it. We would also appreciate if other people are able to contribute further to the package. We would not like to be held accountable for the performance of our package.

**Comment by tejasph** [comment]
I agree with this license because the goal of this project was to learn collaborative development. There is nothing ground-breaking about our current package and I think allowing other people to build on its features would be awesome. For those reasons, I think the MIT license is appropriate.

**Comment by aromatic-toast** [comment]
I agree with all the comments above.
